common/logger/src/security.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//!
+//! The security module gathers security-related logs
+//!
+//! ```
+//! use libra_logger::prelude::*;
+//!
+//! send_struct_log!(
+//!   security_log(
+//!     security_events::INVALID_RETRIEVED_BLOCK)
+//!      .data("some_data", "the data")
+//!   )
+//! );
+//! ```
+//!
+
+use crate::StructuredLogEntry;
+
+/// helper function to create a security log
+pub fn security_log(name: &'static str) -> StructuredLogEntry {
+    StructuredLogEntry::new_named(&name)
+        // set the category to security
+        .security()
+    // set the error description
+}
+
+/// Security events that are possible
+pub mod security_events {
```

Comment:
all of these events are copy/pasted from the security-log crate. I'll have a pass at them in another PR.